scripts/publish-docker-images

```diff
@@ -38,6 +38,7 @@ usage() {
 	echo "  -p  PROFILE	 AWS CLI Profile (default is none)"
 	echo "  -s  BUCKET	  AWS S3 Bucket for staging"
 	echo "  -i  IMAGE	   Docker image name"
+	echo "	-f  FILE PATH	Path to file where manifest JSON output is stored"
```

Comment:
nit: conventionally the flag's option is a single token like `FILE_PATH` when the expectation that its a single arg.